doc/api_assets/style.css

```diff
@@ -488,3 +488,21 @@ th > *:last-child, td > *:last-child {
     font-size: 3.5em;
   }
 }
+
+
+@media print {
+
+  html  {
+    height : auto;
+  }
+
+ #column2.interior {
+   display : none;
+ }
```

Comment:
Something is wrong with indent here, `#column2.interior`, `html`, and `#column1.interior` should be at the same level.